src/coreComponents/dataRepository/DataContext.hpp

```diff
@@ -223,4 +223,23 @@ struct GEOS_FMT_NS::formatter< geos::dataRepository::DataContext > : GEOS_FMT_NS
   }
 };
 
+// The following workaround is needed to fix compilation with NVCC on some machines.
```

Comment:
I think the error message from FMT could be added in the comment so it will be easier if the error shows up again.

Reply:
👍